### PR TITLE
style: add style to app

### PR DESCRIPTION
**Pull request by nottherealalanturing**
## Describe your changes
Setting up project repo with CRA
- Added styles to UI components.

### What kind of change does this PR introduce?
New Feature.

### In which branch do you want the changes to merge?

dev.

### Documentation updated ?

Yes

### Checklist

- [x] No ES Lint Error
- [ ] No Style Lint Error
- [ ] Unit test case update

###Screenshot

Screen Shot 2022-06-09 at 14 56 27

### CODE REVIEW

File: src/components/Book.jsx [hunk 1]
```diff
@@ -1,23 +1,49 @@
...
-      </button>
+    <li className="book-panel">
+      <div className="leftblock">
+        <span className="book-genre">{category}</span>
+        <span className="book-title">{title}</span>
+        <span className="book-author">{author}</span>
+        <span className="leftblockbuttons">
+          <button data-id={id} type="button" className="lbBtn">
+            Comments
+          </button>
```
Comment (thecodechaser) [inline src/components/Book.jsx:15]: - [x] You did a good job on the styles but there are a few issues you need to fix. Please use the correct color for the book buttons(comment, remove, edit) and remove the border around the book buttons, and make sure your design matches the template.

###  Your design

image

### Zeplin design

image

File: src/components/Book.jsx [hunk 2]
```diff
@@ -1,23 +1,49 @@
...
+            <div className="completed">Completed</div>
+          </span>
+        </div>
+
+        <div className="rightblock">
+          <div className="chapter">CURRENT CHAPTER</div>
+          <div className="lesson">Chapter 309</div>
+          <button type="button" className="UpdateData">
+            UPDATE PROGRESS
+          </button>
```
Comment (thecodechaser) [inline src/components/Book.jsx]: - [x] Please remove the border of `'UPDATE PROGRESS'` button and use the correct width and height to make sure the button text is visible on the same line.

### Your design

image

### Zeplin design 

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @nottherealalanturing👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.css [hunk 3]
```diff
@@ -1,7 +1,262 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@900&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@100;300;400;600;700&family=Roboto+Slab:wght@300;400;500;700&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Roboto+Slab&display=swap');
+
```
Comment (BenMukebo) [inline src/index.css:4]: - [OPTIONAL ]  According to the project requirement, Styling must be EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Please make these in order to stick to the original design:
- Kindly add an icon to the right of your navigation bar 😜 
- What if we reduce the space between the `book-genre` and `book-title` 😜 
- Let's add this missing vertical bar to separate the container class div with the rightblock class div

- Your Design:

Screenshot


- Figma design

bookstore

**Review by BenMukebo — APPROVED** [review]
<h1>Status: Approve  :champagne:  :+1: :confetti_ball: 

### **Hi @nottherealalanturing**, 

### **Congratulations! 🎉 Project Approved ✅**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 


### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [OPTIONANL] Please update your Readme, currently some sections are missing,- I would  recommend adding the following: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).
```
- Don't forget to add a link to the issue page, please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.

Cheers :clinking_glasses:  and Happy coding!!! :dancers:💻  

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please ping me @BenMukebo when you comment so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**